region/brinstar/green.json

```diff
@@ -2291,27 +2339,24 @@
                   "name": "Freeze the Beetoms",
                   "notable": false,
                   "requires": [
-                    "Morph",
                     "Ice"
                   ]
                 },
                 {
                   "name": "Tank the Damage",
                   "notable": false,
                   "requires": [
-                    "Morph",
                     {"enemyDamage": {
                       "enemy": "Beetom",
                       "type": "contact",
-                      "hits": 15
+                      "hits": 25
                     }}
                   ]
                 },
                 {
                   "name": "Dodge the Beetoms",
                   "notable": true,
                   "requires": [
-                    "Morph",
```

Comment:
I'm not sure how to dodge them even with morph.

Reply:
Also I forsee notable strats having a required note, or a uniquetech that itself has a note.

Reply:
That's a good call. I dont think unique tech should be notable, since the tech could perfectly describe the strat in that instance. 

It is fairly important that there is a way to get from 4->6 with nothing except that tech, otherwise another node will need to be added - 6 but with no Beetoms on Samus. This strat is a lot easier with the i frames, and because of the farm, it should be equivalent to just the tech. I will add a description of how to do it.